### ISSUE TITLE
creation and update times of elements

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/PermaplanT#1168]
### Tasks

DRAFT issue, will be updated

To have some transparency creation and update times of elements should be added for all elements:

 - [ ] for plants see #476
 - [ ] for drawing layer see #1165

**Comment by horenso** [comment]
A few comments from @chr-schr and me:

1. We suggest that `modification_time` and `creation_time` are non-null and are the same as creation when a new plant is placed on the map (like in Windows).

2. For initial values in the migration we propose:
  - `map.owner_id` for `planting.creation_id`s and `planting.modification_id`s
  - `map.creation_date` for `planting.creation_time` and `planting.modification_id`

**Comment by markus2330** [comment]
1-3: fantastic suggestions, I agree to all of them.

4. no, creator=owner, we need to introduce one term in the glossary and should consistently use that term. Changing to which term is less work?

**Comment by horenso** [comment]
@markus2330 When should map.modified_by and modified_at change? Whenever any api on the map changed something, right? So when we CUD new planting the map should update `map.modified_by` and `modified_at`, change the drawing etc...

**Comment by markus2330** [comment]
Exactly, so some guard needs to be added in every modifying operation. Especially the map date modified date gets modified virtually everywhere a write operation happens.
